### PR TITLE
Update focus styles

### PR DESCRIPTION
**Pull request by ricardoom**
This covers the new focus styles for the header yeah.

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -12,20 +12,28 @@
...
   }
 
   // Accessibility outline rule
-  a, button, input {
-    &:focus, &::-moz-focus-inner {
-      outline: 3px auto #3b99fc;
+  a,
+   button,
+   input {
+    // &:focus,
```
Comment (ktp242) [inline src/styles/main.scss]: Can we delete the old rules here? Will we need it later?

File: src/styles/components/_desktopSearch.scss [hunk 2]
```diff
@@ -116,25 +116,28 @@
...
-        font-style: normal;
         font-size: 22px;
-        background-color: #FFF;
-        color: #000;
-        min-height: 33px;
+        font-style: normal;
+        font-weight: 400;
         height: 100%;
-        padding: 10px 0 10px 10px;
+        line-height: normal !important;
```
Comment (ktp242) [inline src/styles/components/_desktopSearch.scss]: Any other way to avoid using important here?

File: src/styles/components/_desktopSearch.scss [hunk 3]
```diff
@@ -188,26 +191,32 @@
...
       button {
-        position: absolute;
-        top: 67px;
-        right: 0px;
+        @include border-radius(50%);
         background-color: transparent;
         background: transparent;
         border: 2px solid #FFF;
+        height: 72px;
+        line-height: normal !important;
```
Comment (ktp242) [inline src/styles/components/_desktopSearch.scss]: Any other way to avoid using important here?

**Comment by EdwinGuzman** [comment]
You're missing the dist folder/files. I think the correct command is `npm run babel-build`.

**Comment by ktp242** [comment]
Looks good. Can you also update the change log and the version?

**Comment by EdwinGuzman** [comment]
@ricardoom i think all the files were removed again?

File: package.json [hunk 4]
```diff
@@ -64,12 +65,12 @@
   },
   "devDependencies": {
     "axios-mock-adapter": "1.7.1",
-    "babel-cli": "6.11.4",
-    "babel-core": "6.26.0",
+    "babel-cli": "^6.26.0",
```
Comment (EdwinGuzman) [inline package.json]: Remove the carets please.

File: src/components/EmailSubscription/EmailSubscription.js [hunk 5]
```diff
@@ -27,9 +27,8 @@ const styles = {
     border: '2px solid #FFF',
     color: '#FFF',
     fontFamily: 'Kievit-Book',
-    fontSize: '14px',
+    fontSize: '0.875em',
```
Comment (EdwinGuzman) [inline src/components/EmailSubscription/EmailSubscription.js]: If you want to get fancier, this `'0.875em'` value can go in a variable.

File: src/styles/components/_skipNav.scss [hunk 6]
```diff
@@ -24,16 +24,18 @@
...
-      top: 10px;
+      left: auto;
+      outline: 0;
       overflow: visible;
+      top: 10px;
+      width: auto;
+
       @include min-screen(769px) {
-        top: auto;
+         top: auto;
```
Comment (EdwinGuzman) [inline src/styles/components/_skipNav.scss]: extra    space  ?

File: src/styles/utils/_colors.scss [hunk 7]
```diff
@@ -1,7 +1,16 @@
 $libraryRed: #E32B31 !default;
 $libraryBlue: #1B7FA7 !default;
 $BG_MENU_SEARCH: #1B7FA7;
+
 $nypl-teal: #07818d;
+$nypl-orange: #ffb81d;
```
Comment (EdwinGuzman) [inline src/styles/utils/_colors.scss]: These are not in the Design Toolkit?

File: src/components/EmailSubscription/EmailSubscription.js (hunk 5 above)
Comment (ricardoom) [reply src/components/EmailSubscription/EmailSubscription.js]: not sure I want to get fancier at this point in the game... sprint is over for me today.

File: src/styles/components/_skipNav.scss (hunk 6 above)
Comment (ricardoom) [reply src/styles/components/_skipNav.scss]: removed

File: src/styles/utils/_colors.scss (hunk 7 above)
Comment (ricardoom) [reply src/styles/utils/_colors.scss]: they are... 
see:
https://github.com/NYPL/design-toolkit/blob/f21d84d7fa4c32fa994f2b06ee2285b0033274e1/sass/_colors.scss#L41

and 

https://github.com/NYPL/design-toolkit/blob/f21d84d7fa4c32fa994f2b06ee2285b0033274e1/sass/_colors.scss#L26

respectively. Is there an issue here?

File: package.json (hunk 4 above)
Comment (ricardoom) [reply package.json]: gone

File: src/styles/utils/_colors.scss (hunk 7 above)
Comment (EdwinGuzman) [reply src/styles/utils/_colors.scss]: If they're in the DT, then they don't need to be re-declared here?

**Comment by EdwinGuzman** [comment]
Love the new focus style in the header. I'm not sure how big these issues are but the bottom focus outline does not show up when you focus on a tab button that is opened. Examples:

screen shot 2018-06-22 at 11 49 05 am
screen shot 2018-06-22 at 11 49 20 am
If they're not problems then it's good to go.

**Comment by ktp242** [comment]
I will approve it! Looks good. But yeah let us know how you think about the issue Edwin pointed.

File: src/styles/utils/_colors.scss (hunk 7 above)
Comment (ricardoom) [reply src/styles/utils/_colors.scss]: 🤔 this application doesnt use the DTK ¿que no?

**Comment by ricardoom** [comment]
Regarding @EdwinGuzman's point about the `:focus` ring being overlapped by the dropdown below it, I'm not super crazy about it. However, the reasons for the ring(s) to be cut off is due to absolute positioning and z-indexing, I feel like a can of worms is in the opening there and will definitely hold this up. So I'll add an issue and I can take a stab at fixing down the line.

File: src/styles/utils/_colors.scss (hunk 7 above)
Comment (ricardoom) [reply src/styles/utils/_colors.scss]: @EdwinGuzman I think I'll need a short tutorial on how exactly dgx-header uses the DTK...

Comment (EdwinGuzman) [reply src/styles/utils/_colors.scss]: It doesn't until you added it just now. So I believe those variables `$nypl-orange` should be available without having to re-declare them.

**Comment by EdwinGuzman** [comment]
Cool. Double approved.

**Pull request merged**